MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -11,6 +11,11 @@ static partial class GamePad
     {
         internal static bool Back;
 
+        private static int PlatformGetMaxIndex()
+        {
+            return 16;
```

Comment:
I just confused the controller index test (done at the common GamePad.cs level) with a test for gamepad being connected or not (currently done within the extended GamePad.PlatformXXX methods).  They are sort of similar... but not really the same.

It would be nice to move the disconnected gamepad path up into the common GamePad.cs code, but that is for a separate PR.